### ISSUE TITLE
scripts loading twice when mpv folder in appdata

### ISSUE THREAD

**Issue by Obegg** [mpv-player/mpv#14438]
### mpv Information

```bash
mpv v0.38.0-556-g95ac3222 Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
 built on Jun 25 2024 04:06:53
libplacebo version: v7.349.0 (v7.349.0-rc1-3-g1fd3c7b-dirty)
FFmpeg version: N-116015-g23531c977
FFmpeg library versions:
   libavcodec      61.8.100
   libavdevice     61.2.100
   libavfilter     10.2.102
   libavformat     61.4.100
   libavutil       59.26.100
   libswresample   5.2.100
   libswscale      8.2.100
```

### Other Information

- Windows version: Windows 10, 19044.4412
- GPU model, driver and version: VM
- Source of mpv: https://github.com/zhongfly/mpv-winbuild/releases/latest
- Introduced in version: don't know

### Reproduction Steps

download mpv
extract to `%appdata%`
rename folder to `mpv`
create `scripts` folder
create a new file inside scripts folder (use anything other than `.lua`)
open mpv

### Expected Behavior

scripts load once

### Actual Behavior

scripts are loaded twice
```
[   0.017][e][cplayer] Can't load unknown script: C:/Users/Test/AppData/Roaming/mpv/scripts/only1script.txt
[   0.017][e][cplayer] Can't load unknown script: C:\Users\Test\AppData\Roaming/mpv/scripts/only1script.txt
```

### Log File

[dups.log](https://github.com/user-attachments/files/15973137/dups.log)

### I carefully read all instruction and confirm that I did the following:

- [X] I tested with the latest mpv version to validate that the issue is not already fixed.
- [X] I provided all required information including system and mpv version.
- [X] I produced the log file with the exact same set of files, parameters, and conditions used in "Reproduction Steps", with the addition of `--log-file=output.txt`.
- [X] I produced the log file while the behaviors described in "Actual Behavior" were actively observed.
- [X] I attached the full, untruncated log file.
- [X] I attached the backtrace in the case of a crash.

**Comment by hooke007** [comment]
https://mpv.io/manual/master/#files-on-windows
>Config files located in the same directory as mpv.exe are loaded with lower priority. Some config files are loaded only once, which means that e.g. of 2 input.conf files located in two config directories, only the one from the directory with higher priority will be loaded.
>
>A third config directory with the lowest priority is the directory named mpv in the same directory as mpv.exe. This used to be the directory with the highest priority, but is now discouraged to use and might be removed in the future.

**Comment by hooke007** [comment]
Two conf dirs in your case.
>[   0.000][d][global] home path: '' -> 'C:\Users\Test\AppData\Roaming/mpv'
[   0.000][d][global] user path: '~~home/' -> 'C:\Users\Test\AppData\Roaming/mpv'

**Comment by hooke007** [comment]
mpv will search for the `/script` near mpv.exe and `/script` in appdata.

**Comment by hooke007** [comment]
There is no reason to install a software in its conf dir.
I do not see a problem here. If you move mpv.exe to anywhere else and keep the same script both in two `/script` , it will be loaded "twice" too.

**Comment by idMysteries** [comment]
The same problem, mpv had to be moved to Program Files. Now the scripts are loaded once.
It was quite convenient to have mpv.exe in appdata, I could immediately copy everything with configs.
